From 173ff211097edd788a0cd38c5b4d8e6aba93f86a Mon Sep 17 00:00:00 2001
From: Nathan Chancellor <natechancellor@gmail.com>
Date: Fri, 19 Oct 2018 11:30:41 -0700
Subject: [PATCH 26/26] pstore: Fix annotation on declaration of pstore_exit_fs

Clang warns (trimmed for brevity):

fs/pstore/inode.c:504:6: error: section does not match previous
declaration [-Werror,-Wsection]
void __exit pstore_exit_fs(void)
     ^
fs/pstore/internal.h:42:6: note: previous attribute is here
void __init     pstore_exit_fs(void);
     ^

Fixes: a4fb3d2fefd2 ("pstore: Centralize init/exit routines")
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 fs/pstore/internal.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/pstore/internal.h b/fs/pstore/internal.h
index 98bb4fa6e6a7..7062ea4bc57c 100644
--- a/fs/pstore/internal.h
+++ b/fs/pstore/internal.h
@@ -39,6 +39,6 @@ extern void	pstore_record_init(struct pstore_record *record,
 
 /* Called during pstore init/exit. */
 int __init	pstore_init_fs(void);
-void __init	pstore_exit_fs(void);
+void __exit	pstore_exit_fs(void);
 
 #endif
-- 
2.19.1

